week-4/InClass/C-more-complex-objects/exercise-3.js

```diff
@@ -32,18 +32,29 @@ let parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
-
-}
-
-// returns an array of the owners' email addresses of the two houses
-function getEmailAddresses(house1, house2) {
-
-}
-
-// returns the address for the cheapest house out of the two
-function getCheapestAddress(house1, house2) {
-
-}
+  // return house.currentOwner.firstName + " " + house.currentOwner.lastName
+  return `${house.currentOwner.firstName} ${house.currentOwner.lastName}`

```

Comment:
Very good choosing to use interpolation instead of string concatenation. 
It would be great if you delete unneeded comments after you are happy with the results.